### PR TITLE
Add AllJoyn capabilities to Security Camera

### PR DESCRIPTION
**Pull request by bethoma**

### CODE REVIEW

File: SecuritySystemUWP/SecuritySystemUWP/Package.appxmanifest [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <Logo>Assets\StoreLogo.png</Logo>
   </Properties>
   <Dependencies>
-    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.10240.0" MaxVersionTested="10.0.10240.0" />
+    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.10240.0" MaxVersionTested="10.0.10581.0" />
```
Comment (apdutta) [inline SecuritySystemUWP/SecuritySystemUWP/Package.appxmanifest]: I think we should leave this as 240 for now.

File: SecuritySystemUWP/SecuritySystemUWP/AllJoynManager.cs [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+            {
+                this.newestImage = file;
+                this.producer.EmitLastCaptureFileNameChanged();
+                this.producer.EmitLastCaptureTimeChanged();
+            }
+        }
+
+        public IAsyncOperation<SecuritySystemGetLastUploadTimeResult> GetLastUploadTimeAsync(AllJoynMessageInfo info)
+        {
+            return Task.Run(() =>
```
Comment (ooeygui) [inline SecuritySystemUWP/SecuritySystemUWP/AllJoynManager.cs]: I like this pattern.

**Comment by ooeygui** [comment]
:+1:

**Comment by apdutta** [comment]
:+1:

**Pull request merged**